integration/test/metrics/metrics_test.go

```diff
@@ -0,0 +1,60 @@
+// +build k8srequired
+
+package metrics
+
+import (
+	"context"
+	"fmt"
+	"k8s.io/client-go/rest"
+	"testing"
+	"time"
+
+	"github.com/giantswarm/e2esetup/chart/env"
+
+	"github.com/giantswarm/kubernetes-metrics-server/integration/templates"
+)
+
+const (
+	metricsAPIEndpoint = "/apis/metrics.k8s.io/v1beta1"
+)
+
+// TestMetrics ensures that deployed metrics-server chart exposes node-metrics
+// via Kubernetes API extension.
+func TestMetrics(t *testing.T) {
+	ctx := context.Background()
+
+	// Install resource
+	err := r.InstallResource(chartName, templates.MetricsServerValues, fmt.Sprintf("%s-%s", env.CircleSHA(), testName))
```

Comment:
A teardown for this resource is missing.

Reply:
Isn't that handled in main test setup?

Reply:
We usually add a teardown file on the setup package to remove common resources created during setup, see https://github.com/giantswarm/aws-operator/blob/master/integration/setup/teardown.go. If one test creates specific resources it should take care of removing them once it has finished, if not those would be left behind.